### DISCUSSION TITLE
Presenting mulicolored Text

### DISCUSSION THREAD

**Discussion by drei97** [AvaloniaUI/Avalonia#14629] [Q&A]
Hi,

I would like to use a control which is able to present text, but multicolored. Some parts should be white, others red or green, etc.

As far as I can see, the TextPresenter is only able to have one foreground color. If you change property Foreground, the whole text is recolored in the same color.

I read about a RichTextBlock as a candidate, but this control seems not to exist anymore.

Any suggestions?

**Comment by jp2masa** [comment]
You can use inlines with `TextBlock`. The `RichTextBlock` functionality was moved to `TextBlock` in #9245.

**Reply by timunie** [reply]
Yes, exactly. Or SelectableTextBlock if one wants to add selection support.
